zookeeper-server/src/main/java/org/apache/zookeeper/server/NettyServerCnxn.java

```diff
@@ -528,7 +528,7 @@ private void receiveMessage(ByteBuf message) {
                         ZooKeeperServer zks = this.zkServer;
                         if (zks == null || !zks.isRunning()) {
                             LOG.info("Closing connection to {} because the server is not ready",
-                                    getRemoteSocketAddress());
+                                getRemoteSocketAddress(), zks == null ? "null" : zks.getState());
```

Comment:
maybe "UNKNOWN" is better than "null" here ?

Reply:
I think you are right, UNKOWN is a bit more speaking. Done.

Reply:
super-nit: I think I'd prefer "unknown", by capitalizing it we suggest that UNKNOWN is a variant of the State enum, whereas "unknown" would rather suggest that we were unable to figure out the state.
But again, super-super-nitty comment and I have no strong opinion - might just be my brain is wired weird to think about this :)